bindings/pyroot/pythonizations/test/th1_operators.py

```diff
@@ -10,12 +10,20 @@ class TH1Operators(unittest.TestCase):
     """
 
     # Tests
+    def test_numpy(self):
+        import numpy as np
+
+        x = np.random.normal(loc=0.0, scale=1.0, size=10000)
+        w = np.ones(10000)
+        h1 = ROOT.TH1.FromNumpy(x, w, name="h2", title="h2 title", nbins=100, xmin=-5, xmax=5)
```

Comment:
Maybe we can have w not passed when are all ones, and do:
h1 = ROOT.TH1.FromNumpy(x, weights = w, name =....)